src/LaunchDarkly/TargetRule.php

```diff
@@ -43,18 +43,27 @@ public function matchTarget($user) {
                 break;
             case "lastName":
                 $u_value = $user->getLastName();
+                break;
+            case "anonymous":
+                $u_value = $user->getAnonymous();
                 break;                
-            default: 
+            default:
                 $custom = $user->getCustom();
```

Comment:
This logic looked completely broken before. Modifying one of our existing test cases revealed the problem.